packages/api/src/api/api.controller.ts

```diff
@@ -38,11 +48,11 @@ export class ApiController {
   @ApiExcludeEndpoint()
   @Get("api")
   @UseFilters(ApiExceptionFilter)
-  public async apiHandler(
+  public async apiGetHandler(
     @Req() request: Request,
     @Next() next: NextFunction,
-    @Query("module", new ParseModulePipe()) module: ApiModule,
     @Query(new ParseActionPipe()) action: string,
+    @Query("module", new ParseModulePipe()) module: ApiModule,
```

Comment:
Change the order so module pipe works before action pipe. Otherwise even if the module is not found, error about action will be returned.